pol-core/pol/multi/boat.cpp

```diff
@@ -1606,9 +1609,11 @@ void UBoat::readProperties( Clib::ConfigElem& elem )
   rescan_components();
 
   regself();  // do this after our x,y are known.
-  // consider throwing if starting position isn't passable.
+              // consider throwing if starting position isn't passable.
 
-  Core::start_script( "misc/boat", make_boatref( this ) );
+  this->process(
+      Core::start_script( Core::ScriptDef( "misc/boat", nullptr ), make_boatref( this ) ) );
+  this->process()->attached_item_ = this;
```

Comment:
start_script can fail and would then return a nullptr, so this line will crash. check is needed